internal/character/clara/trace.go

```diff
@@ -0,0 +1,64 @@
+package clara
+
+import (
+	"github.com/simimpact/srsim/pkg/engine/event"
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/engine/prop"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+// A2:
+// When attacked, this character has a 35.00 % fixed chance to remove a debuff placed on them.
+// A4:
+// The chance to resist Crowd Control Debuffs increases by 35.00 %.
+// A6:
+// Increases Svarog's Counter DMG by 30.00 %.
+
+const (
+	A2 key.Modifier = "clara-a2"
+	A4 key.Modifier = "clara-a4"
+	A6 key.Modifier = "clara-a6"
+)
+
+func init() {
+	modifier.Register(A2, modifier.Config{
+		Listeners: modifier.Listeners{
+			OnBeforeBeingAttacked: func(mod *modifier.Instance, e event.AttackStart) {
+				if mod.Engine().Rand().Float32() < 0.35 {
+					mod.Engine().DispelStatus(mod.Owner(), info.Dispel{
+						Status: model.StatusType_STATUS_DEBUFF,
+						Order:  model.DispelOrder_LAST_ADDED,
+						Count:  1,
+					})
+				}
+			},
+		},
+	})
+}
+
+func (c *char) initTraces() {
+	if c.info.Traces["1107101"] {
```

Comment:
with https://github.com/simimpact/srsim/pull/134 merged, regenerate `data.go` and change trace ids to the new normalized value